en/ui-system/components/editor/scrollbar.md

```diff
@@ -6,6 +6,8 @@ The ScrollBar allows the user to scroll by dragging a sliding block. It's a bit
 
 Click the __Add Component__ button at the bottom of the __Inspector__ panel and select __UI/ScrollBar__ to add the ScrollBar component to the node.
 
+Api can refer to [ScrollBar API](https://docs.cocos.com/creator3d/api/en/classes/ui.scrollbar.html).
```

Comment:
Api can refer to [ScrollBar API](https://docs.cocos.com/creator3d/api/en/classes/ui.scrollbar.html).
-->
Please refer to the [ScrollBar API](https://docs.cocos.com/creator3d/api/en/classes/ui.scrollbar.html).